repolint.json

```diff
@@ -0,0 +1,54 @@
+{
+    "axioms": {
+      "linguist":"language",
+      "licensee":"license",
+      "packagers":"packager"
+    },
+    "rules": {
+      "all": {
+        "apache-license-file:file-contents": ["error", {"files": ["LICENSE*"], "content": "Apache License.*Version 2.0", "fail-on-non-existent": true}],
+        "code-of-conduct-file:file-contents": ["error", {"files": ["CODE_OF_CONDUCT*"], "content": "https://wiki.hyperledger.org/community/hyperledger-project-code-of-conduct", "fail-on-non-existent": true}],
+        "security-file-matches:file-contents": ["error", {"files": ["SECURITY.md"], "content": "https://wiki.hyperledger.org/display/SEC/Defect.Response", "fail-on-non-existent": true}],
+        "readme-file-exists:file-existence": ["error", {"files": ["README.md", "README"]}],
+        "readme-references-license:file-contents": ["error", {"files": ["README.md", "README"], "content": "license", "flags": "i"}],
+        "maintainers-file-exists:file-existence": ["error", {"files": ["MAINTAINERS.md", "MAINTAINERS.rst"]}],
+        "contributing-file-exists:file-existence": ["error", {"files": ["CONTRIBUTING.md"]}],
+        "changelog-file-exists:file-existence": ["error", {"files": ["CHANGELOG.md"]}],
+        "test-directory-exists:directory-existence": ["error", {"directories": ["**/test*", "**/specs", "**/**_test.go"], "nocase": true}],
+        "integrates-with-ci:file-existence": [
+            "error",
+            {
+              "files": [
+                "circle.yml", 
+                ".circleci/config.yml", 
+                "ci/azure-pipelines.yml",
+                ".ci/azure-pipelines.yml",
+                "Jenkinsfile",
+                "Jenkinsfile.ci",
+                "Jenkinsfile.cd"
+              ]
+            }
+          ],
+
```

Comment:
We should change the `integrates-with-ci` check to only include `ci/azure-pipelines.yml` this is what we decided as our canonical location in Fabric for the file, so we would want the linter